### ISSUE TITLE
Component library navigator is all black on Android

### ISSUE THREAD

**Issue by joshuayoes** [infinitered/ignite#2426]
### Describe the bug

Reproduce:
1. Start a new ignite app as of [463d703](https://github.com/infinitered/ignite/commit/463d7030333e41238b61d733b5148b764e0e744c)
2. `yarn start`
3. `yarn android`
4. Open in simulator

![Screenshot_1682377843](https://user-images.githubusercontent.com/37849890/234136121-314e26ba-f9b9-4871-bbe0-f0ddea6d0f54.png)

### Ignite version

8.6.1

### Additional info

```
System
  platform           darwin                                                  
  arch               arm64                                                   
  cpu                10 cores      Apple M1 Max                              
  directory          boilerplate   /Users/joshuayoes/Code/ignite/boilerplate 

JavaScript (and globally-installed packages)
  node                               18.7.0                                    /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/node 
  npm                                7.24.2                                    /Users/joshuayoes/Code/ignite/node_modules/.bin/npm     
    @githubnext/github-copilot-cli   0.1.21                                                                                            
    detox-cli                        19.0.0                                                                                            
    eas-cli                          2.5.1                                                                                             
    expo-cli                         6.0.6                                                                                             
    ignite-cli                       8.6.1                                                                                             
    openai-chatgpt                   0.0.7                                                                                             
  yarn                               1.22.19                                   /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/yarn 
  pnpm                               7.5.2                                     /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/pnpm 
                                     <no pnpm global package info available>                                                           
  expo                               47.0.14                                   managed                                                 

Ignite
  ignite-cli         8.6.1        /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/.npm/bin/ignite 
  ignite src         src          /Users/joshuayoes/Code/ignite/src                              

Android
  java               11.0.16.1    /Users/joshuayoes/.asdf/shims/java    
  android home       -            /Users/joshuayoes/Library/Android/sdk 

iOS
  xcode              14.2       
  cocoapods          1.12.0       /Users/joshuayoes/.asdf/shims/pod 

Tools
  git                git version 2.39.1   /opt/homebrew/bin/git
```

**Comment by frankcalise** [comment]
Interesting, I don't see this in 8.6.2?

There is a fix for this in the upcoming SDK 48 / RN 0.71, maybe you updated your gesture handler package (where DrawerLayout comes from)?

https://github.com/infinitered/ignite/pull/2377/files#diff-da048c568444cfb67c891221c61dd167f7be4ff580442542a8ed143668413b52

**Comment by joshuayoes** [comment]
I was messing around a lot with my pods and node_modules, so it may be from that. I can take a look again at this.

**Comment by joshuayoes** [comment]
I was hoping to get this merged soon since it helps with getting Ignite compatible with Fabric https://github.com/infinitered/ignite/pull/2377

**Comment by frankcalise** [comment]
> I was hoping to get this merged soon since it helps with getting Ignite compatible with Fabric #2377

Yeah I'm getting there, have to put back a react native bootsplash template change now that it got fixed for 0.71.x

**Comment by joshuayoes** [comment]
This is working now in https://github.com/infinitered/ignite/pull/2377
![Screenshot_1682518496](https://user-images.githubusercontent.com/37849890/234603831-7135e61b-8a08-4db2-9099-1f62cccb5f89.png)
